persistence/non-durable/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Non-durable persistence
+summary: Non-durable persistence (previously known as In-Memory persistence) stores data in a non-durable manner
+component: NonDurablePersistence
+reviewed: 2019-12-17
+redirects:
+- nservicebus/persistence/in-memory
+---
+
+partial: noteinmemory
+
+Some scenarios require an non-durable persistence such as the development environment or a lightweight client not interested in durability across restarts:
+
+snippet: ConfiguringNonDurable
+
+DANGER: All information stored is discarded when the process ends.
+
+NOTE: The [Delayed Retries](/nservicebus/recoverability/#delayed-retries) mechanism uses the [timeout manager](/nservicebus/messaging/timeout-manager.md) when a transport does not natively support delayed delivery. 
```

Comment:
Isn't the NOTE part of DANGER? I think either NOTE should be part of DANGER as a whole section or to just remove NOTE:

Reply:
I don't see how the note should be part of danger. And I still feel like it makes sense as a note. But I can see how it appears to be confusing just because they are grouped together. What if we move the danger note above the snippet?